exec/java-exec/src/main/java/org/apache/drill/exec/rpc/control/ControlConnection.java

```diff
@@ -110,7 +110,9 @@ public boolean equals(Object obj) {
 
   public void shutdownIfClient() {
     if (bus.isClient()) {
-      Closeables.closeQuietly(bus);
+      try {
+        Closeables.close(bus, true);
+      } catch (Exception e) { /* do nothing */ }
```

Comment:
Any particular reason you added this ? is it safe to just ignore runtime exceptions here ?